osu.Game/Screens/Select/FilterQueryParser.cs

```diff
@@ -69,7 +69,7 @@ private static bool tryParseKeywordCriteria(FilterCriteria criteria, string key,
                     return TryUpdateCriteriaRange(ref criteria.BeatDivisor, op, value, tryParseInt);
 
                 case "status":
-                    return TryUpdateCriteriaRange(ref criteria.OnlineStatus, op, value, tryParseEnum);
+                    return TryUpdateArrayRange(ref criteria.OnlineStatus, op, value, tryParseEnum);
```

Comment:
The problem with this implementation is that it breaks a behaviour that was there before and actually worked on stable too - namely _range_ queries with status. Something like `ranked>=r` previously worked (and showed all ranked, approved, qualified, and loved) beatmaps - now it does nothing.

Mind you I'm not sure that many people actually used it, and it wasn't even exercised in the tests, but I do think it is a better choice to keep that working because people get _really_ attached to small features like that.

Reply:
Hmm...should I then implement support for all operators and make the "OptionalArray" more like an OptionalSet? That would mean no dupes and can support ranges (I _think_, will have to experiment)